src/content/h1/guides/map-making/blender-level-guide/level-creation-exporting/readme.md

```diff
@@ -0,0 +1,159 @@
+```.alert
+This guide assumes you have already gone through [level creation - beginner][level-creation-beginner] and [level creation - advanced][level-creation-advanced].
+```
+
+# Introduction
+Now that we've created a level, we can start going through the necessary steps for export. This page will describe all the options you can find in the export menu for exporting a level to .JMS
+
+If there are any images that you find difficult to read then try opening the image in a new tab to view it in full resolution.
+
+# Finding the export menu
+We will be starting by finding the export menu for our Blender exporter. Find the File menu in the toolbar on the top left and open it to reveal the contents. In the menu you will see a few options but the only option relevant to you is the menu option labeled *Export*. To export our geometry as a .JMS file you will want to select the option labeled *Halo Jointed Model Skeleton (.jms)*:
+
+![A nice export menu so thoughtfully provided by our benefactors](A.png)
+
+An export menu should appear on your screen after clicking *Halo Jointed Model Skeleton (.jms)*. It should look something like this:
+
+![Plenty of options for you to mess with. Lets dig in!](B.png)
+
+Lets break down each of the options found in the menu
+
+## Game version
+The *Game Version* box only has a single option:
+
+* Game Version - Dropdown
+	* Halo CE
+	* Halo 2 Vista
+	* Halo 2 MCC
+	* Halo 3 MCC
+		
+For this instance we will be setting *Game Version* to *Halo CE*.
+
+![Why wait for the future when you can let it come to you! This feature will let you quickly export for any title](C.png)
+
+## File Details
+A CE specific box that will let us set the filename for our JMS from the menu easily without having to rename the file manually. Using either of these options will overwrite whatever the user inputs as the filename:
+
+* Permutation - String
+	* Type in a name for the model permutation here. String will default to "unnamed" if LOD is used but permutation is not filled in. See the following page for details on permutations purpose in Halo. [GBXmodel permutations section][gbxmodel#permutations]
+	
```

Comment:
There shouldn't be empty lines in the middle of a list, as it causes the list to be split into two separate lists. The indented sub-list is also not really a list but more an explanation of their parent. We should also avoid phrases like "see the following X" or "click here" and instead prefer to integrate links within our sentences to keep things concise. I think this would be better written as:

* **Permutation - String**: Type in a name for the model permutation here. This string will default to "unnamed" if _LOD_ is used but permutation is not filled in. See the [GBXmodel permutations section][gbxmodel#permutations] for details on permutations' purpose in Halo.
* **LOD - Dropdown**: Select a [level of detail][gbxmodel#level-of-detail] to use for the file here. If the LOD is set to _none_ but permutation is filled in then LOD will default to "Super High".